src/main/java/mate/academy/springboot/model/dto/ProductRequestDto.java

```diff
@@ -0,0 +1,10 @@
+package mate.academy.springboot.model.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
```

Comment:
I think setters and getters would be enough for dto. 
JFYI after java 14 you can use records instead of such dto